src/lexer.coffee

```diff
@@ -793,6 +845,7 @@ exports.Lexer = class Lexer
         last_line:    lastToken[2].last_line
         last_column:  lastToken[2].last_column
       ]
+      lparen[2] = lparen.origin[2]
```

Comment:
The `lparen` token was missing location data, though the data was right there on its `origin` property, so I copy the data up. I need the location data on `lparen` so that `addDataToNode` can match it up with the comments that belong on it.